tools/plugins/com.liferay.ide.portlet.ui/src/com/liferay/ide/portlet/ui/editor/PluginPackageGeneralSection.java

```diff
@@ -102,6 +103,17 @@ public boolean isEditable()
         return true;
     }
 
+    @Override
+    public void commit( boolean onSave )
+    {
+        if( validation() )
```

Comment:
once you call validate, if it has an error then you can set the message on the form with the validation message.